frontend/add-client/contact-information.component.tsx

```diff
@@ -89,14 +89,14 @@ export default function ContactInformation(props: StepComponentProps) {
         </div>
         <div>
           <label>
-            <span>State</span>
-            <StateSelect state={state} setState={setState} />
+            <span>City</span>
+            <CityInput state={state} city={city} setCity={setCity} />
```

Comment:
This change shouldn't be in this PR, since it's relevant to #52, where I commented about how we might not want to do this.